frontend/src/js/constants/Constants.js

```diff
@@ -107,15 +108,149 @@ const JOYRIDE_STEPS = [
-            '</br><span style="color: blue;">Colormap Editor</span> will open a window for creating, editing, and applying colormaps.'
+            '</br><span style="color: blue;">Colormap Editor</span> will open a window for creating, editing, and applying colormaps.',
+            '</br><span style="color: purple;">Export</span> allows you to export/save the plot. NOT Implemented.',
+            '</br><span style="color: orange;">Calculator</span> allows you to create a new variable based on',
+            ' calculations performed on another variable.'
         ),
         selector: ".tools-container",
         position: "right",
         type: "click"
+    },
+    {
+        title: "Calculator",
+        text: "This button opens the calculator tool! Cool right?! Yeah it's cool :p",
+        selector: "#open-calculator-button",
+        position: "right",
+        type: "hover"
     }
 ];
 
+const CALCULATOR_STEPS = [
+    {
+        title: 'Calculator Tour',
+        text: 'This is a tour of the calculator and how to use it!',
+        selector: '#calculator-main',
+        position: 'left',
+        type: 'click'
+    },
+    {
+        title: 'Variables List',
+        text: "All the available variables that you've loaded are listed here.",
+        selector: '#calc-variable-region',
+        position: 'right',
+        type: 'click'
+    },
+    {
+        title: 'Derived Variable',
+        text: "Enter the name of the new variable you wish to derive here.",
+        selector: '#new-variable-name',
+        position: 'left',
+        type: 'click'
+    },
+    {
+        title: 'Calculation Area',
+        text: 'The calculation to be performed will be expressed here.'.concat(
+            '<br />To enter a variable, drag it outside of this area and within the green rectangle that appears.'),
+        selector: '#calculation',
+        position: 'right',
+        type: 'click'
+    },
+    {
+        title: 'Calculator Operations',
+        text: "The calculator supports TWO operands (constants or variables) and ONE operator, when deriving a new variable.".concat(
+            "However you can use previously derived variables to perform further calculations."),
+        selector: '#calc-button-region',
+        position: 'bottom-right',
+        type: 'hover'
+    },
+    {
+        title: 'Regrid Operations',
+        text: "Clicking the 'Regrid' button once you have entered a variable will perform a regrid operation.".concat(
+            "<br />The drop-down on the 'Regrid' button lets you specify what kind of regrid operation to perform.",
+            '<br /><span style="color: blue;">Linear</span> is the default regrid operation.'
+        ),
+        selector: '#regrid',
+        position: 'bottom-right',
+        type: 'hover'
+    },
+    {
+        title: 'Complete Calculation',
+        text: "When you are done setting the calculation click 'Enter' to perform the calculation and have it saved as a new derived variable.",
```

Comment:
"When you are done setting the calculation click the 'Enter' button or simply press 'Enter' to perform the calculation and have it saved as a new derived variable. You can use it right away to derive yet another variable."